modules/adding-tab-pods-page.adoc

```diff
@@ -57,4 +91,4 @@ export default function ExampleTab() {
 ----
 
 .Verification
-* Visit a *Pod* page to view the added tab.
+* Visit a *Pod* page to view the added tab.
```

Comment:
similar comment as Sam's, it looks like we also should document or add additional resource link here to tell use how to run the dynamic plugin before taking `Visit a Pod page to view the added tab` action

Reply:
or else, we should 
1. Deploy the initial console-plugin-template
2. Make your own changes
3. Verify new changes

Reply:
I have added an additional resources link